src/erlcloud_waf.erl

```diff
@@ -0,0 +1,1191 @@
+-module(erlcloud_waf).
+-author('pavel@alertlogic.com').
+
+-include_lib("erlcloud_aws.hrl").
+-include_lib("include/erlcloud_waf.hrl").
```

Comment:
Should file inclusion be one of `-include("x.hrl").`, `-include("../include/x.hrl").` and `-include_lib("erlcloud/include/x.hrl").`?